dev/com.ibm.ws.ejbcontainer.remote_fat/publish/servers/com.ibm.ws.ejbcontainer.remote.fat.SecureRemoteServer/server.xml

```diff
@@ -0,0 +1,44 @@
+<server>
+    <featureManager>
+        <feature>appSecurity-2.0</feature>
+        <feature>ejbHome-3.2</feature>
+        <feature>ejbRemote-3.2</feature>
+        <feature>servlet-3.1</feature>
+        <feature>componenttest-1.0</feature>
+    </featureManager>
+
+    <include location="../fatTestCommon.xml" />
+
+    <iiopEndpoint id="defaultIiopEndpoint" iiopPort="${bvt.prop.IIOP}" >
+        <iiopsOptions iiopsPort="${bvt.prop.IIOP.secure}"  sslRef="defaultSSLConfig"/>
+    </iiopEndpoint>
+
+    <orb id="defaultOrb">
+        <serverPolicy.csiv2>
+            <layers>
+                <attributeLayer identityAssertionEnabled="true" trustedIdentities="*" /> 
+                <authenticationLayer establishTrustInClient="Supported" mechanisms="GSSUP"/>
+                <transportLayer sslRef="defaultSSLConfig"/>
+            </layers>
+        </serverPolicy.csiv2>
+        <clientPolicy.csiv2>
+            <layers>
+                <attributeLayer identityAssertionEnabled="true" trustedIdentity="j2ee" trustedPassword="j2ee"/>
```

Comment:
Is this doing anything anymore?  I see that you've switched the user to "bob" elsewhere.

Reply:
@brideck Thanks, that piece of configuration is actually not needed for what is being tested; it has been removed.